Apps/W1/HybridGP/app/app.json

```diff
@@ -29,7 +29,13 @@
                              "name":  "Sales and Inventory Forecast",
                              "publisher":  "Microsoft",
                              "version":  "22.3.0.0"
-                         }
+                        },
```

Comment:
Is this a free extension? Does this require additional setup for which we need to provide documentation? Do we have plans to update the GP Migration documentation to note this new requirement\dependency for the GP extension?

Reply:
Based on the documentation I read, a Microsoft Exchange license is the only requirement. We may learn more by using it with the PTE in real customer environments. I should add a sentence above the list of email addresses that describe the requirements. I won't for this PR until we learn more. I don't think this feature will be submitted to MSFT until it's battle tested with the PTE.